internal/pkg/model/schema.go

```diff
@@ -333,6 +322,23 @@ type PolicyLeader struct {
 	Timestamp string `json:"@timestamp,omitempty"`
 }
 
+// PolicyOutput holds the needed data to manage the output API keys
+type PolicyOutput struct {
```

Comment:
I like this struct, but I think we should add `Type string `json:"type"` to the struct. See above for the reason.